_pages/a-pairing-session-template.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: default
+title: A Pairing Session Template
+permalink: /template
+meta: A template to help guide your pair programming session
+---
+
+{% include article-top.html %}
+
+Here's a template for your next pairing session:
+
+```
+1. [ ] Agree on the high-level goal out loud.
+2. [ ] Break the work into a handful of tasks.
+3. [ ] Come up with a preliminary ordering for those tasks.
+4. [ ] Decide your driver/navigator swapping strategy.
+5. [ ] Configure git to share credit.
+6. [ ] Eliminate distractions.
+7. [ ] Work.
+8. [ ] Analyze the session.
+```
```

Comment:
Not sure if it's intentional, but I noticed most of the check list items are phrased differently than the headlines below. Might be worth unifying them for clarity.